src/universal/components/OutcomeCard/OutcomeCardFooter.js

```diff
@@ -1,158 +1,125 @@
     isArchived,
     isProject,
+    outcome,
     owner,
+    showTeam,
+    handleStatusClick,
+    styles,
     toggleAssignMenu,
-    handleStatusClick
   } = props;
-  let avatarBlockStyle = {};
-
-  // BUTTONS
-  // --------
-  let buttonStyles = styles.buttonBase;
-  const buttonOptions = [styles.buttonBase];
-
   // AVATAR
   // -------
   const avatarImage = owner.picture;
-  const avatarName = owner.preferredName;
+  const avatarName = showTeam ? outcome.team.name : owner.preferredName;
   // TODO: Set avatarTeam style when showing team instead of owner (on UserDashboard)
   // const avatarStyles = combineStyles(styles.avatar, styles.avatarTeam);
-  const avatarStyles = styles.avatar;
   const menuHintStyle = cardHasHover ? faStyle : {visibility: 'hidden', ...faStyle};
   let buttonIcon = hasOpenStatusMenu ? 'times' : 'wrench';
   if (isArchived) buttonIcon = 'reply';
+  const showFully = (hasOpenStatusMenu || cardHasHover || isArchived);
 
-  if (!isProject) { buttonOptions.push(styles.actionButton); }
-
-  if (hasOpenStatusMenu || cardHasHover) {
-    if (isProject) {
-      buttonOptions.push(styles.projectButtonShowFully);
-    } else {
-      buttonOptions.push(styles.actionButtonShowFully);
-    }
-  }
-
-  if (isArchived) {
-    buttonOptions.push(styles.projectButtonShowFully);
-    avatarBlockStyle = {
-      cursor: 'not-allowed',
-      opacity: 1
-    };
-  }
-
-  buttonStyles = combineStyles(...buttonOptions);
-
+  const avatarBlockStyle = css(
+    styles.avatarBlock,
+    isArchived && styles.avatarBlockArchived
+  );
+  const buttonStyles = css(
+    styles.buttonBase,
+    !isProject && styles.actionButton,
+    showFully && (isProject ? styles.projectButtonShowFully : styles.actionButtonShowFully)
+  );
   return (
-    <div className={styles.root}>
-      <div className={styles.avatarLayout}>
-        <div className={styles.avatarBlock} onClick={toggleAssignMenu} style={avatarBlockStyle}>
-          <img
-            alt={avatarName}
-            className={avatarStyles}
-            src={avatarImage}
-          />
-          <div className={styles.name}>{avatarName}</div>
+    <div className={css(styles.root)}>
+      <div className={css(styles.avatarLayout)}>
+        <div className={avatarBlockStyle} onClick={toggleAssignMenu}>
+          {!showTeam &&
+            <img
+              alt={avatarName}
+              className={css(styles.avatar)}
+              src={avatarImage}
+            />
+          }
+          <div className={css(styles.name)}>{avatarName}</div>
           {!isArchived &&
             <FontAwesome
-              className={styles.menuHint}
+              className={css(styles.menuHint)}
               name="ellipsis-v"
               style={menuHintStyle}
             />
           }
         </div>
       </div>
-      <div className={styles.buttonBlock}>
+      <div className={css(styles.buttonBlock)}>
         <button className={buttonStyles} onClick={handleStatusClick}>
```

Comment:
Ah yeah, we had that, but seeing your face 50 times was kinda weird, so me n Terry figured we'd show the team, but let them change the team member. Since team ID is built into the immutable primary key, wed essentially delete and recreate. Changing teams also means that we can't easily see the change history of a project due to permissions. I'm cool with whatever